nestkernel/connection_manager.cpp

```diff
@@ -924,8 +924,9 @@ nest::ConnectionManager::get_connections( std::deque< ConnectionID >& connectome
       std::vector< index > target_device_node_ids;
       split_to_neuron_device_vectors_( tid, target, target_neuron_node_ids, target_device_node_ids );
 
+      // Getting regular connections, if they exist.
```

Comment:
This also includes connections from devices, is that something in addition to "regular connections", or does it all fall under one?